src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -828,14 +828,22 @@ private async Task<string> CreatePullRequestAsync(List<UpdateAssetsParameters> u
             }
         }
 
-        private void UpdatePRDescriptionDueConfigFiles(List<GitFile> commitedFiles, StringBuilder globalJsonSection)
+        public static void UpdatePRDescriptionDueConfigFiles(List<GitFile> committedFiles, StringBuilder globalJsonSection)
         {
-            GitFile globalJsonFile = commitedFiles?.
+            GitFile globalJsonFile = committedFiles?.
                 Where(gf => gf.FilePath.Equals("global.json", StringComparison.OrdinalIgnoreCase)).
                 FirstOrDefault();
 
             if (globalJsonFile != null)
```

Comment:
Could this be globalJsonFile?.Metadata != null?

Reply:
Yeap